lib/audit/moab_to_catalog.rb

```diff
@@ -1,14 +1,19 @@
 ##
 # Method that will check the a single moab service disk for the existence of moabs in postgres database
 class MoabToCatalog
-  def self.check_moab_to_catalog_existence(storage_dir)
+  def self.check_moab_to_catalog_existence(storage_dir, expect_to_create=false)
```

Comment:
method takes expect_to_create but does nothing with it yet?  (not even pass it thru?)

Reply:
Yeah I need to add the use case for that in the rspec test - not sure if this is the best way to do this though, we can probably come up with a better way so we don't have to set expect_to_create=true every time we want to create an object.. @jmartin-sul ?

Reply:
i think the idea is that `expect_to_create` is usually `false` (because it's being used by the regular inventory process), but that it's `true` when we're populating the catalog for the first time (in which case, it'd be nice to not get warned 1.5 million times when we expect to be creating all the catalog entries on that particular invocation).

Reply:
I actually think we will want to create specific code to create the catalog for the first time, and use the "create" method.  This is partly b/c we may add to this code to populate the TCR, since looping through each moab takes some CPU effort.

Reply:
F2F group discussion -- we agreed that there will be a "seed db" entry point into code that calls "create" and a separate "M2C existence check" entry point into code.  @jmartin-sul created appropriate tickets for that work, so this is now 👍 in the context of this PR.